src/model/Ticket.java

```diff
@@ -77,4 +112,41 @@ private void validateEventBackpackWeightLimit(int backpackWeight) {
         }
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
follow the order:
constants, variables, constructors, setters/getters, equals, hashCode, toString, other methods